### DISCUSSION TITLE
How to create a snippet for a new snippet block

### DISCUSSION THREAD

**Discussion by jonas-frey** [draivin/hsnips#58] [Q&A] [answered]
I want to have a snippet that expands to 

```
snippet $1 A
$2
endsnippet
```
where $1 and $2 are tab stops.

I imagine for this to work I have to escape at least the `endsnippet` somehow? How can I do that?

Secondly, in which file to put the resulting snippet? Would that be `hsnips.hsnips`?

Thanks!

**Comment by draivin** [comment, marked as answer]
Currently there is no way to do that in plain text, as you said, we would need an escape sequence. But it is still doable by using code interpolation, just place the `endsnippet` segment inside a code interpolation that returns just that string!

As to the second question, I believe that `hsnips.hsnips` is the correct file.

**Reply by jonas-frey** [reply]
Thanks! I confirm that 
```
snippet snp A
snippet $1 A
$2
``rv = "endsnippet"``

endsnippet
```
in `hsnips.hsnips` works!
